editor/editor_settings.h

```diff
@@ -102,6 +102,7 @@ class EditorSettings : public Resource {
 	HashMap<String, List<Ref<InputEvent>>> builtin_action_overrides;
 
 	Vector<String> favorites;
+	HashMap<StringName, PackedStringArray> favorite_properties;
```

Comment:
I wonder if it shouldn't be String instead of StringName. You seem to be doing some String conversions, and also some keys can be script paths.